python/graphstorm/sagemaker/sagemaker_train.py

```diff
@@ -141,6 +141,8 @@ def run_train(args, unknownargs):
             S3 location of training yaml file.
         model_artifact_s3: str
             S3 location to store the model artifacts.
+        node_mappings_s3: str
```

Comment:
We do not generate node embeddings or do prediction in training.
So we do not need to change sagemaker_train.py